ci/test/05_before_script.sh

```diff
@@ -10,7 +10,7 @@ export LC_ALL=C.UTF-8
 if [ "$CI_OS_NAME" == "macos" ]; then
   echo > "${HOME}/Library/Application Support/Bitcoin"
 else
-  CI_EXEC echo \> \$HOME/.bitcoin
+  CI_EXEC_ROOT echo \> \$HOME/.bitcoin
```

Comment:
This looks wrong? Maybe just create both to be save?

Reply:
not sure why i originally put this here :thinking: i removed it and everything still worked

Reply:
The idea is that the main directory should not be read/created when running the tests. This is achieved by creating a file with that name. Now that there are two users, it seems safer to create two files.